client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -35,11 +36,18 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 
 		return map( without( steps, STEPS.SUMMARY ), stepName => {
 			const isCompleted = get( stepsCompleted, stepName ) === true;
+			const isPending = stepName === STEPS.WOOCOMMERCE && this.props.isWooCommerceInstallPending;
 			const className = classNames( 'steps__summary-entry', isCompleted ? 'completed' : 'todo' );
 
+			const indicator = isPending ? (
+				<Spinner />
+			) : (
+				<Gridicon icon={ isCompleted ? 'checkmark' : 'cross' } size={ 18 } />
+			);
+
 			return (
 				<div key={ stepName } className={ className }>
-					<Gridicon icon={ isCompleted ? 'checkmark' : 'cross' } size={ 18 } />
+					{ indicator }
```

Comment:
why not inline this? seems like it's simple enough that extracting it is harder to follow